templates_for_code_gen/filter.cc.handlebars

```diff
@@ -57,8 +57,8 @@ public:
   }
   bool onConfigure(size_t /* configuration_size */) override;
 
-  std::string_view getWorkloadName() { return workload_name_; }
-{{#each udfs}}
+  StringView getWorkloadName() { return workload_name_; }
```

Comment:
This should be std::string_view, not StringView.

Reply:
Got it;  that's changed.